themes/_default.yaml

```diff
@@ -23,8 +23,11 @@ generic:
     accent-darker: &accent-darker   '#0B4848'
     accent-dark: &accent-dark       '#177D7D'
     accent: &accent                 '#2A9595'
+    accent-border: &accent-border   '#E6BB00'
```

Comment:
Not what? :| the accent border was not there I added it lol.

Didn't even knew that we had any specific system to choose the colors 😮 .

Then you'll have to tell me how to select them ;-) (And I have to admit that the calendar colors look a little bit bad now 😄 )